packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/components/Questions/components/CheckBoxQuestion/CheckBoxQuestion.js

```diff
@@ -0,0 +1,66 @@
+import { Box } from 'grommet'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import CheckBoxOption from './components/CheckBoxOption'
+
+export default function CheckBoxQuestion (props) {
```

Comment:
Probably been staring at too long, but maybe `CheckBoxQuestion` and `RadioQuestion` should be `CheckBoxOptions` and `RadioOptions` as individual question labels are rendered in `Questions` and really the components just represent the options?

Reply:
Or maybe `CheckBoxInputs` and `RadioInputs`? Not really required for this PR though.